book/docs/classes/OpusPool/Methods/buildMintTransaction.md

```diff
@@ -0,0 +1,30 @@
+### buildMintTransaction
+
+▸ **buildMintTransaction**(`params`): `Promise`\<[`MintTransactionData`](../../../interfaces/MintTransactionData.md)\>
+
+Generates a mint transaction to mint osTokens from the chosen vault
+
+#### Parameters
+
+| Name | Type | Description |
+| :------ | :------ | :------ |
+| `params` | `Object` | Parameters for building the transaction |
+| `params.shares` | `bigint` | An amount of shares to be minted |
+| `params.vault` | \`0x$\{string}\` | A vault address |
+| `params.referrer?` | \`0x$\{string}\` | An address of the referrer. Optional |
```

Comment:
I think it would be useful to expand on what `referrer` means and what rewards will go to this address